app/models/user.rb

```diff
@@ -3,6 +3,10 @@ class User < ApplicationRecord
   has_many :comments, class_name: 'Comment'
   has_many :likes, class_name: 'Like'
 
+  # validations
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than: -1 }
```

Comment:
Please **PostsCounter must be an integer greater than or equal to zero**. Kindly Refer to the Project requirements